students/joel_rhode/lesson_5/assignment/tests/test_database.py

```diff
@@ -0,0 +1,67 @@
+"""
+Module for testing of database.py, HP Norton mongoDB implementation.
+"""
+
+from unittest import TestCase
+from unittest.mock import patch
+
+from database import (import_data, read_csv, write_many_to_database, show_available_products,
+                      show_rentals)
+import database
+from test_files.expected_lists import customer_list, product_list, rental_list
```

Comment:
Your test file is currently setup to be called from the _assignment_ directory, so this import will fail.